clippy_dev/src/update_lints.rs

```diff
@@ -194,7 +192,6 @@ pub fn rename(old_name: &str, new_name: &str, uplift: bool) {
             let name = f.path().file_name();
             let ext = f.path().extension();
             (ext == Some(OsStr::new("rs")) || ext == Some(OsStr::new("fixed")))
-                && name != Some(OsStr::new("rename.rs"))
```

Comment:
This changed in a previous PR. Renamed lints were moved into the same file as deprecated lints.

Reply:
I assume with previous PR you mean #13180. That removed `renamed_lints.rs`, right, but the separate ui test `rename.rs` still exists? Maybe I don't understand what this exclusion is for in the first place. I'd assumed it's because the ui test and the list of deprecations/renames require special handling and are regenerated, and they still are, so I would think that both conditions here are still needed?

Sorry for all the questions, I'd like to properly understand what's going on here 😅